### ISSUE TITLE
How can I make the latents in data_latents and latents_ in edits_sg? 

### ISSUE THREAD

**Issue by naeun0620** [RameenAbdal/CLIP2StyleGAN#4]
Hi, I have a question. It would be really helpful if you answer me. 

I am wondering how to make the `*.npy` s in `data_latents` and `latents_` folders.

For example, If I want to make `latents_/beard_glass.npy` and `data_latents/men.npy` for edit_type 'beard', what should I save? 

Is the normalized `B` in https://github.com/RameenAbdal/CLIP2StyleGAN/blob/main/optimize_dir.py#L85 can be `latents_/beard_glass.npy`?    

Also, how can I make `data_latents/men.npy`? Can I just get all the men images and extract features like the code below? 
```
   image_data = natsorted(glob.glob(args.path))
   with torch.no_grad():
      for i in range(len(image_data)):
         if i == 0:
            i_x = preprocess(Image.open(image_data[i])).unsqueeze(0).to(device)
            image_features_pred = model.encode_image(i_x)
         else:
            i_x = preprocess(Image.open(image_data[i])).unsqueeze(0).to(device)
            image_features_pred = torch.cat((image_features_pred, model.encode_image(i_x)),dim = 0)
```

Thanks,

**Comment by Harry-zzh** [comment]
> Hi, I have a question. It would be really helpful if you answer me.
> 
> I am wondering how to make the `*.npy` s in `data_latents` and `latents_` folders.
> 
> For example, If I want to make `latents_/beard_glass.npy` and `data_latents/men.npy` for edit_type 'beard', what should I save?
> 
> Is the normalized `B` in https://github.com/RameenAbdal/CLIP2StyleGAN/blob/main/optimize_dir.py#L85 can be `latents_/beard_glass.npy`?
> 
> Also, how can I make `data_latents/men.npy`? Can I just get all the men images and extract features like the code below?
> 
> ```
>    image_data = natsorted(glob.glob(args.path))
>    with torch.no_grad():
>       for i in range(len(image_data)):
>          if i == 0:
>             i_x = preprocess(Image.open(image_data[i])).unsqueeze(0).to(device)
>             image_features_pred = model.encode_image(i_x)
>          else:
>             i_x = preprocess(Image.open(image_data[i])).unsqueeze(0).to(device)
>             image_features_pred = torch.cat((image_features_pred, model.encode_image(i_x)),dim = 0)
> ```
> 
> Thanks,

I have the same question, have you solved it ?
